### PR TITLE
Create 2024-08-22_hosting_rshiny_applications.md

### PR DESCRIPTION
**Pull request by hrhotz**

### CODE REVIEW

**Comment by hrhotz** [comment]
ping @csoneson @federicomarini 
you will find more info here: https://galaxyproject.org/news/2024-07-08-galaxy-social/

File: posts/2024-08-22_hosting_rshiny_applications.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ mastodon-eu-freiburg:
+  - galaxyproject@mstdn.science
+  - csoneson@fosstodon.org
+  - federicomarini@genomic.social
+  - bgruening@fosstodon.org
+  - hrhotz@genomic.social 
+ bluesky-galaxyproject:
+  - galaxyproject.bsky.social
+ matrix-eu-announce:
+  - bgruening:matrix.org
```
Comment (bgruening) [inline posts/2024-08-22_hosting_rshiny_applications.md:18]: That one should be removed :-) it was just for demonstration...

Comment (hrhotz) [reply posts/2024-08-22_hosting_rshiny_applications.md]: removed
(I just used the template 😉 )

**Review by bgruening — APPROVED** [review]

**Comment by arash77** [comment]
Unfortunately, there was a problem with sending the post to Matrix and that caused the program to fail to send it to LinkedIn too but Mastodon and Bluesky are good:
https://bsky.app/profile/galaxyproject.bsky.social/post/3l2clhtpzgr2y
https://xn--baw-joa.social/@galaxyfreiburg/113005548404643709

It seems that it is related to the matrix API key that is not working anymore!
with merging [the Enhance Post Failure Handling PR](https://github.com/usegalaxy-eu/galaxy-social/pull/53) we will handle these kinds of situations better.
Also, I had to add entries to `processed_files` manually so the file will not posted again!

**Pull request merged**